prevent-broker-scale-down-if-contains-partition .md

```diff
@@ -0,0 +1,49 @@
+
+# Prevent scale down of the brokers if they contain replicas
+
+This proposal is about adding the mechanism for preventing the scale down of the brokers in the cluster if one or more replicas are hosted on it.
+In the future, when we introduce the automatic re-balancing when scaling up/down the Kafka Cluster ([proposal](https://github.com/strimzi/proposals/blob/main/035-rebalance-types-scaling-brokers.md)), this mechanism would help to prevent any data loss when we are scaling down the brokers when they have some partitions assigned to them.
+
+## Current situation
+
+Currently, when the number of Kafka replicas is decreased in the Kafka CR the StatefulSet will shut down the highest numbered brokers and there is no check to see if these brokers contain any topic-partition replicas.
+The [documentation](https://strimzi.io/docs/operators/latest/configuring.html#scaling-clusters-str) recommends that all topic-partition replicas are reassigned before doing this scaling action, which means that if someone tries to scale down without re-assigning the brokers, it can cause data loss.
```

Comment:
```suggestion
The [documentation](https://strimzi.io/docs/operators/latest/configuring.html#scaling-clusters-str) recommends that all topic-partition replicas are reassigned before doing this scaling action, which means that if someone tries to scale down without re-assigning the brokers, it can cause availability issues or data loss.
```